modal/sandbox.py

```diff
@@ -562,6 +562,23 @@ async def from_id(sandbox_id: str, client: Optional[_Client] = None) -> "_Sandbo
 
         return obj
 
+    async def get_tags(self, client: Optional[_Client] = None):
+        """Fetches any tags (key-value pairs) attached to this Sandbox from the server."""
+        environment_name = _get_environment_name()
```

Comment:
😖 I know this is also mostly following what we already have but the environment scoping for sandbox tags seems super broken. The "current environment" in the location you call `.get_tags()` should not factor into what tags you retrieve for an existing Sandbox.